mlflow/utils/search_utils.py

```diff
@@ -115,7 +115,7 @@ def case_sensitive_mysql_eq(value):
             )
 
         def case_sensitive_mysql_ne(value):
-            return sa.text(f"{col} != :value OR BINARY {col} != :value").bindparams(
+            return sa.text(f"({col} != :value OR BINARY {col} != :value)").bindparams(
```

Comment:
Nice catch.

Reply:
@harupy See this failure test: (corresponding commit: 1596ab9)
https://github.com/mlflow/mlflow/runs/7647679932?check_suite_focus=true#step:4:606

The searching result contains duplicated items. I am still thinking what causes this.

Reply:
oh I got it. The wrong `OR` operator causes tag table filter out multiple rows, and it joins with the model table so it generate duplicated results.